.github/workflows/get_dockerfile_path.py

```diff
@@ -0,0 +1,64 @@
+import os
+import yaml
+
+def extract_organization_name(pull_request_description):
+    try:
+        data = yaml.safe_load(pull_request_description)
+        model_details = data.get('Model Details', {})
+        org_name = model_details.get('Organization Name', None)
+        return org_name
+    except Exception as e:
+        print(f"Error while parsing YAML: {e}")
+        return None
+
+def get_pull_request_description():
```

Comment:
I think I saw this function elsewhere in the same PR. We should pull out all common function into a separate module and `import` them.

Reply:
I agree. Having everything in the same module and then just importing makes it less redundant and cleaner.